model-archiver/README.md

```diff
@@ -58,7 +58,7 @@ torch-model-archiver --model-name densenet161 --version 1.0 --model-file example
 $ torch-model-archiver -h
 usage: torch-model-archiver [-h] --model-name MODEL_NAME  --version MODEL_VERSION_NUMBER
                       --model-file MODEL_FILE_PATH --serialized-file MODEL_SERIALIZED_PATH
-                      --handler HANDLER [--runtime {python,python2,python3}]
+                      --handler HANDLER [--runtime {python,python3}]
```

Comment:
For a period of time we could just map python3 to python with a deprecation warning and instruction to select executable through config.